### PR TITLE
Fix api tests

### PR DESCRIPTION
**Pull request by agennadi**
# Description

Refactored API tests, added a test for api/polygons. 
Removed a trailing space in "prettierrc." (It caused problems with git on Windows)

## Type of changes
- [x] Bugfix
- [ ] Chore
- [x] New Feature

## Testing
- [ ] I added automated tests
- [x] I think tests are unnecessary

### CODE REVIEW

**Comment by agennadi** [comment]
Hi Mack, I rebased the branch. Let me know if there are any comments.

**Comment by agennadi** [comment]
@mackcooper1408 Hi Mack, i deleted the duplicate files :)

File: api/index.py [hunk 1]
```diff
@@ -7,13 +7,16 @@
 """
 # TODO:
 # Decide: 
-# - whether old polygons should be kept #53
```
Comment (mackcooper1408) [inline api/index.py:10]: let's keep these in, they reference the GH issue numbers

**Review by mackcooper1408 — APPROVED** [review]
looks great! small comment to actually leave the Issue numbers in that big TODO

**Pull request merged**